WordPress/gutenberg
# Delete widget editor redundant css for more menu

## Description
This is a small PR that removes some unused CSS. It's a follow-up to #39084.

The More Menu is now implemented in the interface package which already has this CSS (https://github.com/WordPress/gutenberg/blob/trunk/packages/interface/src/components/more-menu-dropdown/style.scss), so the widget editor doesn't need to duplicate it.

## Testing Instructions
1. Open the widget editor
2. Open the editor More Menu (three dots in the top right corner)
3. It should still look and behave the same.

## Types of changes
Code quality
